pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -3210,7 +3211,26 @@ let
 
       ms-vscode-remote.remote-ssh = callPackage ./ms-vscode-remote.remote-ssh { };
 
-      ms-vsliveshare.vsliveshare = callPackage ./ms-vsliveshare.vsliveshare { };
+      ms-vsliveshare.vsliveshare = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "vsliveshare";
+          publisher = "ms-vsliveshare";
+          version = "1.0.5918";
+          hash = "sha256-Tk0mKydUF8M7l7NC9wEA7t2rzJWy/mq4/HvIHI2/ldQ=";
+        };
+
+        postPatch = ''
+          substituteInPlace extension.js \
+            --replace-fail '"xsel"' '"${xsel}/bin/xsel"'
+        '';
+
+        meta = {
+          description = "Real-time collaborative development for VS Code";
```

Comment:
`changelog` is missing if you care to add. This is used by the bot when upgrading the package.

Reply:
Added that, thanks for the suggestion!